### ISSUE TITLE
SDK dashboard components shared tab states

### ISSUE THREAD

**Issue by WiNloSt** [metabase/metabase#49345]
### Describe the bug

When using this markup

```tsx
    <MetabaseProvider config={config}>
      <Title order={2}>{'<InteractiveDashboard />'}</Title>
      <InteractiveDashboard dashboardId={8} withDownloads />
      <Divider my="lg" />

      <Title order={2}>{'<StaticDashboard />'}</Title>
      <StaticDashboard dashboardId={8} />
      <Divider my="lg" />

      <Title order={2}>{'<EditableDashboard />'}</Title>
      <EditableDashboard dashboardId={8} />
      <Divider my="lg" />
    </MetabaseProvider>
```

The dashboard tab state is shared among the 3 dashboard components. There might be other states shared unintentionally.

![Image](https://github.com/user-attachments/assets/21554be9-afed-4531-9e45-32d32ee40d7b)

### To Reproduce

1. Use the markup above and embed a dashboard with more than 1 tab
2. Run the user app, and change tab on one of the component, you'll notice that all dashboard components also change the tab.

### Expected behavior

The state isn't shared since they're rendered separately.

### Information about your Metabase installation

- 

### Severity

annoying

**Comment by WiNloSt** [comment]
@deniskaber I'm not aware of the latest discussion, is there a doc or anything I can look at so I could refresh my memory?

But as you mentioned, if this is not supported, then maybe I could rephrase this bug into a feature request like support more than 1 dashboard in a single provider? What do you thinK?

**Comment by heypoom** [comment]
FYI, we do have this notice in our `README.md` in NPM currently:

```md
- Embedding multiple instances of interactive dashboards on the same page is not supported.
  - Please use static dashboards if you need to embed multiple dashboards on the same page.
```

We don't explicitly mention the issue with static and interactive dashboards having a shared state though, so we might possibly want to update the README here?

**Comment by WiNloSt** [comment]
> FYI, we do have this notice in our `README.md` in NPM currently:
> 
> - Embedding multiple instances of interactive dashboards on the same page is not supported.
>   - Please use static dashboards if you need to embed multiple dashboards on the same page.
> We don't explicitly mention the issue with static and interactive dashboards having a shared state though, so we might possibly want to update the README here?

@heypoom I think we should update that to reflect the new behavior. Something along the line that only `StaticDashboard` supports multiple instances, the rest (`Interactive*`, and `Editable*`) aren't.

**Comment by heypoom** [comment]
@WiNloSt True. I think the strange behaviour that is worth pointing out is that even using `StaticDashboard` with `InteractiveDashboard` in fact does not work - both dashboard converges to a single shared dashboard id/state.
